en/docs/administer/updating-wso2-api-manager.md

```diff
@@ -7,6 +7,8 @@ The WSO2 in-place updates tool allows you to update your currently used product
 
 For more information, see [Using WSO2 In-Place Updates](https://docs.wso2.com/display/updates100/Using+WSO2+In-Place+Updates)
 
+!!! note
+    WSO2 recommends all users to shift to the new WSO2 Updates 2.0 model to alleviate from future updates challenges. See [WUM/In-place updates decommission plan](https://updates.docs.wso2.com/en/latest/updates/wum-decommission/) for more information.
```

Comment:
```suggestion
    WSO2 recommends all users to shift to the new WSO2 Updates 2.0 model to alleviate from future updates challenges. For more information, see [WUM/In-place updates decommission plan](https://updates.docs.wso2.com/en/latest/updates/wum-decommission/).
```

Reply:
Always tell the you why they need to click the link before telling them what the link is. We have used this concept in the docs right throughout.

[1] https://sites.google.com/a/wso2.com/technical-content-team/writing-tips#TOC-Referring-users-to-other-sources